### ISSUE TITLE
Do not flag line length in multi-line string literals

### ISSUE THREAD

**Issue by JakeWharton** [shyiko/ktlint#262]
When using a multi-line string literal, line length should not be enforced.

Given a line length of 15, the following should not produce an error:
```kotlin
val long = """
This is a line that's longer than 15
"""
```

**Comment by JakeWharton** [comment]
Wow that was fast. Thanks!

**Comment by shyiko** [comment]
Fixed in [0.26.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0260---2018-07-30) (Maven Central sync might take up to 20m) :balloon:
